angular-ngrx-scss/src/app/home/user-gists/user-gists.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
This test file can also be deleted. The component doesn't actually handle any logic, so not much to test here.

- [ ] Can remove this test file